lighthub/item.cpp

```diff
@@ -684,7 +684,7 @@ int Item::Ctrl(short cmd, short n, int *Parameters, boolean send, int subItemN)
             //prescaler = 5 ---> PWM frequency is 30 Hz
             //prescaler = 6 ---> PWM frequency is <20 Hz
             int tval = 7;             // this is 111 in binary and is used as an eraser
-#if defined(__AVR_ATmega2560__)
+#if defined(ARDUINO_ARCH_AVR)
```

Comment:
Wrong aproach - timer here is MEGA2560 specific

Reply:
ARDUINO_ARCH_AVR will be defined when compiling for AVR-based boards
This is standard and common define. So it's much more readable and understandable than __AVR_ATmega2560__

Reply:
since we have only mega2560 based boards in platformio.ini file this question doesn't matter.